cs.html

```diff
@@ -301,6 +302,55 @@ <h2>Methods</h2>
                 <em>Return type:</em> <code><a>QuicSendStream</a></code>
               </div>
             </dd>
+            <dt><dfn><code>readyToSendDatagram</code></dfn></dt>
+            <dd>
+              <p>Returns a promise that will be <a>fulfilled</a> when the QuicTransport can send
```

Comment:
I think the right lingo is "resolved", not "fulfilled"

Reply:
Actually, it looks like the WebRTC spec mostly uses "resolved", but sometimes uses "fulfilled", so maybe it doesn't matter.  But let's at least make sure we're consistent in our docs.

Reply:
I originally changed to "fulfilled" after reading these:
- https://stackoverflow.com/questions/29268569/what-is-the-correct-terminology-for-javascript-promises
- https://github.com/domenic/promises-unwrapping/blob/master/docs/states-and-fates.md

I updated the PR back to resolved (for consistency). It might be worth filing an issue to update the language to fulfilled, because from my understanding from those readings is that's the most appropriate word.

Reply:
I made an issue for it:  https://github.com/w3c/webrtc-quic/issues/99

We can continue discussing there, if it's worth doing so.